edit-post/components/layout/style.scss

```diff
@@ -70,6 +70,7 @@
 
 .edit-post-layout__content {
 	position: relative;
+	padding-bottom: 100vh;
```

Comment:
What's the purpose of this change?

Reply:
This could use a comment as well. :) To pad the scroll box if there is not enough content to scroll.